priv/templates/vm.args.eex

```diff
@@ -2,7 +2,7 @@
 -name <%= release_name %>@127.0.0.1
 
 ## Cookie for distributed erlang
--setcookie <%= release_name %>
+-setcookie <%= hd(:io_lib.format("~p", cookie)) %>
```

Comment:
This will fail, `io_lib:format` expects the second argument to be a list. We should just use `String.to_atom/1` here, and ensure that the value provided in the config is converted to an atom if it's not one already.